google-built-opentelemetry-collector/README.md

```diff
@@ -50,7 +50,7 @@ The Google-Built OpenTelemetry Collector is an open-source, production-ready bui
 | resource | [docs](https://www.github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourceprocessor/README.md) |
 | resourcedetection | [docs](https://www.github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor/README.md) |
 | tailsampling | [docs](https://www.github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/tailsamplingprocessor/README.md) |
-| transform | [docs](https://www.github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor/README.md) |
+| transform | [docs](https://www.github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/agenttransformprocessor/README.md) |
```

Comment:
I've gone back and forth on whether I think you should replace the `transform` processor or add a new `agenttransform` processor. Either way this change is wrong (it calls it `transform` but points at an upstream directory that doesn't exist)

Reply:
Removed the READMED updates. I deferred back to calling it only `transformprocessor`.